packages/neuron-wallet/src/services/cells.ts

```diff
@@ -693,20 +695,20 @@ export default class CellsService {
           })
       : CellsService.getLiveOrSentCellByLockArgsMultisigOutput(lockClass))
 
-    const liveCells = cellEntities.filter(c => c.status === OutputStatus.Live)
+    const useCells = enableUseSentCell ? cellEntities : cellEntities.filter(c => c.status === OutputStatus.Live)
```

Comment:
I cannot recall my point exactly. I think my question is 
1. when `enableUseSentCell` is `true`, `useCells` is the group of `sent` and `live` cells;
2. when there are 2 transactions constructed sequentially, will they be built from the same cell group?
---
I think inputs of pending transactions should be treated as dead(treated as dead but not actually dead), by doing so, many conflicts could be avoided.

Reply:
Do you mean only sending one transaction if the user sends two transactions sequentially? I'm afraid not because the last transaction was broadcast to the chain.

Reply:
> Do you mean only sending one transaction if the user sends two transactions sequentially? I'm afraid not because the last transaction was broadcast to the chain.

Not `only sending one transaction if the user sends two transactions sequentially`.

The question is,
1. there's a pending output A in neuron;
2. submit transaction 1, it includes the pending output A as an input;
3. instantly submit transaction 2, will it include the pending output A as an input? If not, which status is output A marked.

Reply:
For https://github.com/nervosnetwork/neuron/pull/2963#discussion_r1427520317.
The inputs sent by Neuron will be marked as `Pending`, then it will not be used in the next transactions. We will only use the `Live` or `Sent` cell.